guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -5,9 +5,10 @@ functional_areas:
   - Cloud
   - Setup
   - Configuration
+redirect_from:
```

Comment:
does this need a redirect for the 2.2 one also?

Reply:
After looking deeper into this, I do not believe we should delete the older versions of this file. Those release notes have nothing to do with the ECE-tools release notes. It is perfectly fine to delete or redirect the offending 2.3 placeholder file, but the others should remain or be archived. They were legit for their particular release AND they are reference in the Magento Commerce release notes.  @jeff-matthews (Copying you because you commented on the original issue.)

Reply:
I moved the redirect directive for the  v2.2 /cloud/release-notes/CloudReleaseNotes.html  to the v2.2 ece-tools release note file (v2.2/cloud/release-notes/cloud-tools.md).

Reply:
I agree that we should not delete https://devdocs.magento.com/guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.html. I'm more concerned with the empty files.

We _should_ add a link to the new release notes in that file though to help guide users who land there from google search.

We should also investigate adding a link to the new release notes to this page as requested in the issue linked to this PR: https://devdocs.magento.com/guides/v2.3/release-notes/bk-release-notes.html.